transaction.go

```diff
@@ -110,7 +110,7 @@ func (t *Tracer) StartTransactionOptions(name, transactionType string, opts Tran
 				// we will scale the sampled transactions.
 				result.SampleRate = 0
 			}
-			sampleRate := round(1000*result.SampleRate) / 1000
+			sampleRate := roundSampleRate(result.SampleRate)
```

Comment:
ah, hadn't noticed this!

Reply:
No worries, I forgot about it when reviewing your PR :smile: